flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing. why not build from source?

Reply:
this is unanswered.

Reply:
The game takes a long time to build from the source code, there are official builds, I don’t see the point in this.

Reply:
If the community sees a real need for ARM support, we may consider it in the future, but for now I think it makes more sense to use the official x86_64 builds.

Reply:
> The game takes a long time to build from the source code,  

this is irrelevant.

> I don’t see the point in this.

aarch64 support.

Reply:
Compiling from source can be time consuming, especially for a large project like this. Given that official builds already exist, it makes more sense to use them. Also, I don't have the hardware to test aarch64 builds, so focusing on official x86_64 builds seems like the most rational approach. If ARM support really becomes important to the community, we may consider it in the future.